misc/semver-checker/modules/semver-checker-core/src/main/java/io/ballerina/semver/checker/comparator/DumbNodeListComparator.java

```diff
@@ -39,9 +40,9 @@
  */
 public class DumbNodeListComparator<T extends Node> implements Comparator {
 
-    protected final List<T> newNodesList;
-    protected final List<T> oldNodesList;
-    protected String nodeKindName;
+    private final List<T> newNodesList;
+    private final List<T> oldNodesList;
```

Comment:
IMO, instead of using the List prefix, pluralizing the variable would work right. WDYT? Like, instead of `newNodesList` just `newNodes`

Reply:
+1. Updated with [c17b9fd](https://github.com/ballerina-platform/ballerina-lang/pull/37482/commits/c17b9fd7282ec2ec3691300c2dfc70bc84c8cdd9)